workflows/kallisto-quant/run-kallisto.nf

```diff
@@ -23,10 +23,9 @@ params.index_path = "${params.ref_dir}/${params.index_dir}/${params.index_name}"
 params.t2g_path = "${params.ref_dir}/${params.annotation_dir}/${params.t2g}"
 params.mito_path = "${params.ref_dir}/${params.annotation_dir}/${params.mitolist}"
 
-
 process kallisto_bus{
   container 'quay.io/biocontainers/kallisto:0.46.2--h4f7b962_1'
-  label 'cpus_8'
```

Comment:
I think we can have multiple labels? https://www.nextflow.io/docs/latest/process.html#label If so, I would prefer that we separate disk and memory/cpu labels. But in this case, since only this module requires more memory, we probably can specify the memory and cpu specifically for this process (the configuration is not shared by other processes).

Reply:
Yes we can, so I could adjust it to have the bigdisk label with memory referenced in the process. Then the config file wouldn't be altered.

Reply:
I have no problem with adding more labels, if we want to go that route too.... We could even separate cpu number from memory, but we just want to be careful about trying too hard to optimize. I think having more processes with similar resource requests is more efficient in general.